shared/utils/prismic.js

```diff
@@ -14,67 +15,74 @@ function linkResolver(doc) {
     case 'articles':
       return '/articles';
     case 'article':
-      return `/articles/${doc.slug || doc.uid}`;
+      return `/articles/${doc.uid}`;
     case 'custom_page':
-      return `/${doc.slug || doc.uid}`;
+      return `/${doc.uid}`;
     default:
       return '/';
   }
 }
 
-function renderAsText(field) {
-  try {
-    return asText(field);
-  } catch (e) {
-    console.warn('unable to render field as text', field);
-  }
+const SHOULD_WARN = true;
 
-  return '';
+function warn(...m) {
+  if (SHOULD_WARN) {
+    console.warn(...m);
+  }
 }
 
-function renderAsComponentTree(field) {
-  if (!field) {
-    console.warn('cannot render undefined field');
-    return null;
+function get(obj, path = undefined, defaultValue = '') {
+  const value = path ? _get(obj, path) : obj;
+
+  if (!value) {
+    return defaultValue;
```

Comment:
Is there a reasen for why we are not using _get(obj, path, defaultValue) instead of the following ?
if (!value) {
    return defaultValue;
}

Reply:
Yes, related to the above about having an option to not get an object.

So support something like this:

```
const foo = 'text';
const result = getString(foo);
```